lmfdb/genus2_curves/web_g2c.py

```diff
@@ -430,9 +431,9 @@ def split_statement(coeffs, labels, condnorms):
             statement += """<br>&nbsp;&nbsp;Elliptic curve <a href=%s>%s</a>""" % (url_for_ec(label), label)
         # Otherwise give defining equation:
         else:
-            statement += """<br>&nbsp;&nbsp;\(y^2 = x^3 - g_4 / 48 x - g_6 / 864\) with<br>\
-            \(g_4 = %s\)<br>\
-            \(g_6 = %s\)<br>\
+            statement += """<br>&nbsp;&nbsp;\\(y^2 = x^3 - g_4 / 48 x - g_6 / 864\\) with<br>\
+            \\(g_4 = %s\\)<br>\
```

Comment:
Shouldn't these trailing backslashes be removed?  It seems like you intended Python line continuation backslashes, but they're not necessary in HTML.

Reply:
Yes (this has been like this for ages, not changed in this PR, but I agree it should be cleaned up and the indentation is wrong in any case). I just fixed this.

Reply:
You can see an example running the modified code at:
https://red.lmfdb.xyz/Genus2Curve/Q/999936/a/999936/1